github/config.go

```diff
@@ -22,6 +22,9 @@ type Config struct {
 	Insecure         bool
 	WriteDelay       time.Duration
 	ReadDelay        time.Duration
+	RetryDelay       time.Duration
+	RetryableErrors  map[int]bool
```

Comment:
This could also be achieved with a `struct` FWIW, though it's not a blocking concern by any means.